src/ModulesList.jsx

```diff
@@ -38,41 +29,34 @@ export default class ModulesList extends React.Component {
     };
   }
 
-  getModules() {
-    let self = this;
-    let $tokenInput  = $('[name=wt-token]');
-    let $urlInput = $('[name=wt-url]');
-    let url = $urlInput.val();
-    let token = $tokenInput.val();
-
-    fetch_modules(token, url)
-    .done((data) => {
-      self.setState({
-        ready: true,
-        node_version: data.node_version,
-        modules: data.modules,
-        limit: 12
-      });
-    }).fail(() => {
-      self.setState({
-        modules: [],
-      });
-    });  
+  componentWillMount() {
+    let $versionDropdown = $('#version-dropdown');
+    $versionDropdown.val(getNodeVersion());
```

Comment:
Submitting with `?node_version=4` query param sets the dropdown, but the results are still for node 8. Perhaps related to inconsistent types?

![image](https://user-images.githubusercontent.com/947526/41950689-37ebdafe-7995-11e8-9aac-b1efa6f3cc80.png)